.zuul.yaml

```diff
@@ -360,48 +239,8 @@
       Run gophercloud acceptance test on the master branch. This runs when any file is patched
       except if it's doc.
     vars:
-      # the list of all available tests can generated by:
-      # find acceptance/openstack -name '*_test.go' -exec dirname {} \; | sort -n | uniq -c | awk '{print $2}'
       acceptance_tests:
         - acceptance/openstack
-        - acceptance/openstack/blockstorage
-        - acceptance/openstack/blockstorage/extensions
-        - acceptance/openstack/blockstorage/v3
-        - acceptance/openstack/compute/v2
-        - acceptance/openstack/container/v1
```

Comment:
Shouldn't we leave in the legacy job services that haven't yet been ported to github action?

From that list, I think we're missing:
- container
- orchestration
- placement
- sharedfilesystems

Reply:
done